model/legacy-services/src/main/java/org/keycloak/exportimport/util/MultipleStepsExportProvider.java

```diff
@@ -84,7 +84,11 @@ protected void runExportImportTask(KeycloakSession session) throws IOException {
                 // Count total number of users
                 if (!exportUsersIntoRealmFile) {
                     usersHolder.totalCount = session.users().getUsersCount(realm, true);
-                    federatedUsersHolder.totalCount = UserStorageUtil.userFederatedStorage(session).getStoredUsersCount(realm);
+                    if (UserStorageUtil.userFederatedStorage(session) != null) {
```

Comment:
Can this be `null`? I see that this is part of `legacy-services` module. Isn't it true that when this module is used legacy storage is used? Is this causing problems in the testsuite?